.github/workflows/release.yml

```diff
@@ -44,8 +44,8 @@ jobs:
 
     - name: "Remove old tags"
       run: |
-          git push --delete origin ${{ env.X }}
-          git push --delete origin ${{ env.X }}.${{ env.Y }}
+        git push --delete origin ${{ env.X }} || true
+        git push --delete origin ${{ env.X }}.${{ env.Y }} || true
 
```

Comment:
This could mask other issues IMO. I would do a more active checking - verify if the tag exists or not, and if it exists, delete it.

For example, if the action is enabled in a repo with some specific tag protection... we could be hitting an error that gets masked because the `` || true`` section swallows anything. For the sake of our end users, I would do the active checking

Reply:
> protected tags

☝ 
I'm gonna change my implementation on the if chance